### PR TITLE
added /people/jp3671!!

### PR DESCRIPTION
**Pull request by drpajama**
Hey it's PJ!! Would you add mine? 👍 👍 👍 👍 👍 👍

### CODE REVIEW

**Comment by drpajama** [comment]
Of course! I will be working on it!!

**Comment by drpajama** [comment]
Done!!!

**Comment by drpajama** [comment]
Done again!

File: people/jp3671.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+last_name: Park
+first_name: Joongheum
+permalink: /people/jp3671/
+---
+
+
+## Academic Interests
+I'm interested in the intersection between informatics and cooking!
+
+### Prediction of Flavor/Tastes of Food from Recipes
```
Comment (mark-velez) [inline people/jp3671.md]: I actually would rather appreciate this :)

**Pull request merged**